includes/admin/forms/class-metabox-form-data.php

```diff
@@ -301,9 +301,9 @@ function get_settings() {
 							'style'         => 'width: 100%',
 							'wrapper_class' => 'give-hidden give-title-prefixes-wrap',
 							'options'       => array(
-								'Mr'   => __( 'Mr', 'give' ),
-								'Mrs'  => __( 'Mrs', 'give' ),
-								'Ms'   => __( 'Ms', 'give' ),
+								'Mr.'   => __( 'Mr.', 'give' ),
```

Comment:
@mehul0810 create a helper function with a filter for default title fixes and use this function here.